test/viewmodels/checkAnswers/sections/firstTransporter/FirstTransporterVatSummarySpec.scala

```diff
@@ -79,21 +70,18 @@ class FirstTransporterVatSummarySpec extends SpecBase with Matchers {
             implicit lazy val request = dataRequest(FakeRequest(), emptyUserAnswers)
 
             FirstTransporterVatSummary.row(showActionLinks = false) mustBe
-              SummaryListRowViewModel(
-                key = messagesForLanguage.cyaLabel,
-                value = Value(Text(messagesForLanguage.notProvided)),
-                actions = Seq()
-              )
+              None
           }
 
           "when there is an answer" in {
```

Comment:
As above, if you refactor out the change links then these duplicate tests won't be needed